jerry-core/ecma/base/ecma-helpers.h

```diff
@@ -44,12 +50,33 @@
 #define ECMA_SET_NON_NULL_POINTER(field, non_compressed_pointer) JMEM_CP_SET_NON_NULL_POINTER (field, \
                                                                                                non_compressed_pointer)
 
+/**
+ * Set value of pointer-tag value so that it will correspond
+ * to specified non_compressed_pointer along with tag
+ */
+#define ECMA_SET_NON_NULL_POINTER_TAG(field, non_compressed_pointer, tag) \
+  JMEM_CP_SET_NON_NULL_POINTER_TAG (field, non_compressed_pointer, tag)
+
 /**
  * Set value of compressed pointer so that it will correspond
  * to specified non_compressed_pointer.
  */
 #define ECMA_SET_POINTER(field, non_compressed_pointer) JMEM_CP_SET_POINTER (field, non_compressed_pointer)
 
+/**
+ * Get value of each tag bit from specified pointer-tag value
+ */
+#define ECMA_GET_FIRST_BIT_FROM_POINTER_TAG(field) JMEM_CP_GET_FIRST_BIT_FROM_POINTER_TAG (field)
+#define ECMA_GET_SECOND_BIT_FROM_POINTER_TAG(field) JMEM_CP_GET_SECOND_BIT_FROM_POINTER_TAG (field)
+#define ECMA_GET_THIRD_BIT_FROM_POINTER_TAG(field) JMEM_CP_GET_THIRD_BIT_FROM_POINTER_TAG (field)
```

Comment:
Unfortunately ou have to add doxygen comments for each macro. Here and below.

Reply:
Thank you for the review. 
But `appveyor` is still not working :(
Is there any way to force the trigger of `appveyor`?